service/web/adminserver_test.go

```diff
@@ -247,3 +249,177 @@ func TestAdminServer_ListProjectCredentials(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, expectedCredProto, p)
 }
+
+func Test_adminServer_CreateProjectCredentials(t *testing.T) {
+	ctx := context.Background()
+	ctrl := gomock.NewController(t)
+	mockCredentialsCore := mocks4.NewMockICore(ctrl)
+	req := &metrov1.ProjectCredentials{ProjectId: "test-project"}
+
+	tests := []struct{ err error }{
```

Comment:
you should include name of test in test struct and run test case like for _, tt := range tests { t.Run(tt.name, func(t *testing.T) {}}
this will log test name if test case is failing